engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -2541,6 +2858,11 @@ static uintptr_t GetExtProcAddress(const char* name, const char* extension_name,
         }
     }
 
+    static uint8_t OpenGLGetNumTextureHandles(HTexture texture)
+    {
+        return texture->m_NumTextureIds;
+    }
+
```

Comment:
Used when binding the texture(s), but we could potentially remove this function and let the GL adapter handle the individual binds. Might be easier